oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -596,19 +596,17 @@ impl Runtime {
     }
 
     // TODO(#970): Improve the documentation.
```

Comment:
@tiziano88 If the modified documentation is good enough, I can remove this line and close #970. WDYT?

Reply:
SG, but would it make sense to actually update https://github.com/project-oak/oak/blob/master/docs/abi.md#wait_on_channels and redirect both this documentation and that on the wasm node implementation to that? Otherwise we still have 3 pieces of text saying essentially the same thing in slightly different ways.